app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -131,6 +131,59 @@ public function form(Form $form): Form
                             ->label('Active')
                             ->hidden(fn (Get $get): bool => blank($get('logo'))),
                     ]),
+                Section::make('Branded Website Links')
+                    ->aside()
+                    ->schema([
+                        Section::make('Support')
+                            ->schema([
+                                Toggle::make('is_support_url_enabled')
+                                    ->label('Enable Support URL')
+                                    ->hintIcon(fn (Toggle $component) => $component->isDisabled() ? 'heroicon-m-lock-closed' : null)
```

Comment:
I don't think this `hintIcon` is needed as this is the toggle itself and there is currently nothing that would cause this to be disabled.